lighthouse-core/report/html/renderer/util.js

```diff
@@ -424,10 +424,24 @@ class Util {
         networkThrottling = Util.i18n.strings.runtimeUnknown;
     }
 
-    // TODO(paulirish): revise Runtime Settings strings: https://github.com/GoogleChrome/lighthouse/pull/11796
-    const deviceEmulation = settings.formFactor === 'mobile'
-      ? Util.i18n.strings.runtimeMobileEmulation
-      : Util.i18n.strings.runtimeDesktopEmulation;
+
+    let deviceEmulation;
+    // COMPAT: remove this fallback handling at Lighthouse v8
```

Comment:
Why remove in v8? Don't we want to keep the renderer backwards-compat indefinitely?

Reply:
Oh, right, there's three possibilities for the old way...

Reply:
```
deviceEmulation = {
  mobile: Util.i18n.strings.runtimeMobileEmulation,
  desktop: Util.i18n.strings.runtimeDesktopEmulation,
  none: Util.i18n.strings.runtimeNoEmulation,
}[settings.formFactor || settings.emulatedFormFactor || 'none']
```

maybe?

Reply:
yes